aireview/pyproject.toml

```diff
@@ -9,11 +9,16 @@ packages = [{ include = "aireview", from = "src" }]
 [tool.poetry.dependencies]
 python = "^3.11"
 requests = "^2.32.3"
-pytz = "^2024.2"
 msrest = "^0.7.1"
 azure-devops = "^7.1.0b4"
```

Comment:
**documentation**: Removed pytz dependency without explanation. If timezone functionality is still needed, consider documenting why this dependency was removed or use Python's built-in zoneinfo (Python 3.9+).